### PR TITLE
chore: Fix minor bugs, provide shim and optimize proguard

### LINKED ISSUE #484: Fix minor bugs and provide shims
Some minor alignment bugs remain in the app along with proguard optimization and shim for debug dependencies

Working on it

### PR DESCRIPTION
**Pull request by iamareebjamal**
- Fix layout alignment issues
- Provide shim for stetho
- Optimize jackson proguard for low app size
- Remove unused script
- Add null check on chart analyser
- Remove null check for ticket analyser
- Move barcode emitter code to appropriate place

Fixes #484

### CODE REVIEW

**Comment by iamareebjamal** [comment]
@mejariamol Please review

File: app/build.gradle [hunk 1]
```diff
@@ -17,11 +17,11 @@ android {
     compileSdkVersion 25
     buildToolsVersion "25.0.2"
     defaultConfig {
-        applicationId "org.fossasia.openevent.app"
+        applicationId "org.fossasia.eventyay"
```
Comment (mejariamol) [inline app/build.gradle]: Aren't we changing package structure in the app according to this?

Comment (iamareebjamal) [inline app/build.gradle]: No

**Comment by iamareebjamal** [comment]
@niranjan94 Please merge

**Pull request merged**